### PR TITLE
💄  ✨  Move action buttons for status to dropdown

### LINKED ISSUE #1489: On Mobile when someone has a long username, the design is getting broken.
**Describe the bug**
On Mobile when someone has a long username, the design is getting broken. Like the Join Button or Likes.

**To Reproduce**

1. Go to to the Website on Mobile
2. Go to the Dashboard
3. See error if someone has a long username

**Expected behavior**
The Buttons should be better designed and organized.

**Screenshots**
grafik
grafik
grafik

**Smartphone (please complete the following information):**

Any Smartphones

### PR DESCRIPTION
**Pull request by HerrLevin**
The possible interactions for a status are cluttering the interface more and more. 
To fix this issue, every interaction except liking has been moved to a dropdown menu.

image

image

Should fix #1489

Preparation for #1384

### CODE REVIEW

**Review by MrKrisKrisu — CHANGES_REQUESTED** [review]
image

I don't really like the new layout as much. 

1. Please add more space between the profile picture and the text
2. Can you put this back in one line?
3. Can you make the dots bigger?

**Comment by HerrLevin** [comment]
@MrKrisKrisu like that? (done)
image

**Review by MrKrisKrisu — APPROVED** [review]
okay for me. Not a fan of the text on the left, but maybe it's just unfamiliar for me.

(i have not looked at the code for this)

File: app/Enum/Business.php [hunk 1]
```diff
@@ -3,9 +3,35 @@
...
+        return match ($this) {
+            self::PRIVATE  => __('stationboard.business.private'),
+            self::BUSINESS => __('stationboard.business.business'),
+            self::COMMUTE  => __('stationboard.business.commute')
+        };
+    }
+
+    public function description(): string {
+        return match ($this) {
+            self::PRIVATE  => '',
```
Comment (MrKrisKrisu) [inline app/Enum/Business.php:32]: should this be an empty string?

File: app/Enum/StatusVisibility.php [hunk 2]
```diff
@@ -3,11 +3,43 @@
...
+    }
+
+    public function title(): string {
+        return match ($this) {
+            self::PUBLIC        => __('status.visibility.0'),
+            self::UNLISTED      => __('status.visibility.1'),
+            self::FOLLOWERS     => __('status.visibility.2'),
+            self::PRIVATE       => __('status.visibility.3'),
+            self::AUTHENTICATED => __('status.visibility.4')
+        };
```
Comment (MrKrisKrisu) [inline app/Enum/StatusVisibility.php:33]:
```suggestion
        return __('status.visibility.' . $this->value);
```

Wouldn't this be easier? (untested)

File: app/Enum/Business.php (hunk 1 above)
Comment (HerrLevin) [reply app/Enum/Business.php:32]: yes. We never had a description for that.

File: app/Enum/StatusVisibility.php (hunk 2 above)
Comment (HerrLevin) [reply app/Enum/StatusVisibility.php:33]: Yes but I'd like to increase readability and maybe change the name of the translation in the future (visibility.0 is very ambiguous)

**Pull request merged**